spec/unit/application/lookup_spec.rb

```diff
@@ -78,7 +78,7 @@
   end
 
 
-  context 'when asked to explain' do
+  context 'when using a proper fixture' do
```

Comment:
As opposed to "files not on disk" ? Please write it from an example point of view. Is this testing files on disk as opposed to something else? Is it supposed to test " proper fixture" as opposed to "improper fixture"?

Te name is supposed to describe *what* is tested (features/functonaity), not *how* it is tested.